docs/addons.md

```diff
@@ -322,6 +322,18 @@ spec:
       enabled: true
 ```
 
+##### Self-provisioned aws-ebs-csi-driver
+
+The following aws-ebs-csi-driver configuration allows provisioning aws-ebs-csi-driver externally. Please note that if you’re using Amazon EBS volumes, you must install the Amazon EBS CSI driver. If the Amazon EBS CSI plugin is not installed, then volume operations will fail.
```

Comment:
I wonder if we should have a comment here about IRSA.

If IRSA is not enabled, the control plane will have the permissions to provision nodes, and the self-managed controllers should be running on the control plane.

If they have IRSA enabled, kOps will create roles and policy for the driver's SA, and users need to add volume and envs themselves. This could be done by also installing the identity webhook addon and annotating the SA. The challenge there is that the role name is likely truncated, so one needs to find the correct role manually.

Reply:
Good idea 👍